core/capabilities/ccip/oraclecreator/bootstrap.go

```diff
@@ -306,61 +484,6 @@ func (d *peerGroupDialer) closeExistingPeerGroups() {
 	d.activeConfigDigests = []cciptypes.Bytes32{}
 }
 
-func (d *peerGroupDialer) createNewPeerGroups() error {
-	activeConfigDigest, candidateConfigDigest := d.rmnHomeReader.GetAllConfigDigests()
-	var configDigests [][32]byte
-
-	if !activeConfigDigest.IsEmpty() {
-		configDigests = append(configDigests, activeConfigDigest)
-	}
-	if !candidateConfigDigest.IsEmpty() {
-		configDigests = append(configDigests, candidateConfigDigest)
-	}
-
-	d.lggr.Infow("Creating new peer groups", "configDigests", configDigests)
-
-	for _, rmnHomeConfigDigest := range configDigests {
-		rmnNodesInfo, err := d.rmnHomeReader.GetRMNNodesInfo(rmnHomeConfigDigest)
-		if err != nil {
-			return fmt.Errorf("get RMN nodes info: %w", err)
-		}
-
-		h := sha256.Sum256(append(d.commitConfigDigest[:], rmnHomeConfigDigest[:]...))
-		genericEndpointConfigDigest := writePrefix(ocr2types.ConfigDigestPrefixCCIPMultiRoleRMNCombo, h)
-
-		peerIDs := make([]string, 0, len(d.oraclePeerIDs))
-		for _, p := range d.oraclePeerIDs {
-			peerIDs = append(peerIDs, p.String())
-		}
-		for _, n := range rmnNodesInfo {
-			peerIDs = append(peerIDs, n.PeerID.String())
-		}
-
-		lggr := d.lggr.With(
-			"genericEndpointConfigDigest", genericEndpointConfigDigest.String(),
-			"peerIDs", peerIDs,
-			"bootstrappers", d.bootstrapLocators,
-		)
-
-		lggr.Infow("Bootstrapper is creating new peer group")
-		peerGroup, err := d.peerGroupFactory.NewPeerGroup(
-			[32]byte(genericEndpointConfigDigest),
-			peerIDs,
-			d.bootstrapLocators,
-		)
-		if err != nil {
-			lggr.Errorw("failed to create new peer group", "err", err)
-			return fmt.Errorf("new peer group: %w", err)
-		}
-		lggr.Infow("Created new peer group successfully")
-
-		d.activePeerGroups = append(d.activePeerGroups, peerGroup)
-		d.activeConfigDigests = append(d.activeConfigDigests, genericEndpointConfigDigest)
-	}
-
-	return nil
-}
-
 func writePrefix(prefix ocr2types.ConfigDigestPrefix, hash cciptypes.Bytes32) cciptypes.Bytes32 {
```

Comment:
Lets create a ticket to extract the common pieces of this code and the code in chainlink-ccip and re-use where possible.

Reply:
done!